src/wolfio.c

```diff
@@ -1462,8 +1498,14 @@ int EmbedOcspLookup(void* ctx, const char* url, int urlSz,
                 ret = wolfIO_HttpProcessResponseOcsp(sfd, ocspRespBuf, httpBuf,
                                                  HTTP_SCRATCH_BUFFER_SIZE, ctx);
             }
-            if (sfd != SOCKET_INVALID)
+            if (sfd != SOCKET_INVALID) {
+            #ifdef FUSION_RTOS
+                int err;
+                CloseSocket(sfd, &err);
```

Comment:
If we don't have a macro for CloseSocket please add one. I think we do and this logic can be done in a single place via a macro.

Reply:
Addressed.